examples/light-switch-app/nrfconnect/README.md

```diff
@@ -545,7 +551,7 @@ To test this example, two devices - a lighting device (eg. [Lighting-App example
 
 To perform the binding process you need a controller which can write binding table to light switch device and write proper ACL to an endpoint lighting device. For example, the [chip-tool for Windows/Linux](../../chip-tool/README.md) can be used as a controller. An ACL should contain information about all clusters which can be called by light-switch. See [chip-tool-guide interacting with ZCL clusters section](../../../docs/guides/chip_tool_guide.md#interacting-with-zcl-clusters) for more information about ACLs.
 
-### Binding using chip-tool for Windows/Linux
+### Binding unicast device using chip-tool for Windows/Linux
```

Comment:
Unicast device doesn't sound correct. I would say "unicast binding". Moreover we don't bind to the whole device, but to the specific endpoint on a device.
```suggestion
### Unicast binding to the remote endpoint using chip-tool for Windows/Linux 
```